orbit/cmd/orbit/orbit.go

```diff
@@ -1551,16 +1551,19 @@ func main() {
 		go sigusrListener(c.String("root-dir"))
 
 		isLinux := runtime.GOOS == "linux"
-		serverHasWebSetup := orbitClient.GetServerCapabilities().Has(fleet.CapabilityWebSetupExperience)
 		setupExperienceNotDisabled := !c.Bool("disable-setup-experience")
-		runSetupExperience := isLinux && serverHasWebSetup && setupExperienceNotDisabled
+		runSetupExperience := isLinux && setupExperienceNotDisabled
 		log.Debug().
 			Bool("isLinux", isLinux).
-			Bool("serverHasSetup", serverHasWebSetup).
 			Bool("notDisabled", setupExperienceNotDisabled).
 			Msg("checking setup experience preflight values")
 
 		openMyDevicePage := func() error {
+			if !c.Bool("fleet-desktop") {
```

Comment:
Added this because `orbit` was panicking when building fleetd without `--fleet-desktop` (`trw.Read` was panicking below because `trw` is `nil` when there's no Fleet Desktop)